src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Solver.hpp

```diff
@@ -114,50 +158,14 @@ namespace AirflowNetwork {
         Array1D<Real64> SUMF;
     };
 
-    extern std::vector<AirProperties> properties;
+    //extern std::vector<AirProperties> properties;
```

Comment:
One more here and there are a bunch of others. We should probably make a pass to clean these out.

Reply:
This one should go, but the other's still serve a purpose for now.